sci-libs/oce/oce-0.18.3.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils eutils check-reqs multilib java-pkg-opt-2
+
+DESCRIPTION="Development platform for CAD/CAE, 3D surface/solid modeling and data exchange"
+HOMEPAGE="https://github.com/tpaviot/oce"
+SRC_URI="https://github.com/tpaviot/oce/archive/OCE-$PV.tar.gz"
+
+LICENSE="|| ( Open-CASCADE-LGPL-2.1-Exception-1.0 LGPL-2.1 )"
+SLOT="${PV}"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="examples freeimage gl2ps openmp +tbb +vtk X"
+REQUIRED_USE="?? ( openmp tbb )"
+
+MY_VTK="vtk-8.1"
```

Comment:
Is there a special reason why this value isn't simply inlined in `DEPEND`?

Reply:
Nope. It should be inlined. Will fix.